src/stores/Annotation/Annotation.js

```diff
@@ -882,7 +882,8 @@ export const Annotation = types
         }
       } else {
         // unselect labels after use, but consider "keep labels selected" settings
-        if (control.type.includes('labels')) self.unselectAll(true);
+        // if (control.type.includes('labels')) self.unselectAll(true);
```

Comment:
are we looking to put that if statement back in at some point? or should be it be removed?